20241009.md

```diff
@@ -0,0 +1,48 @@
+# RISE RP005 QEMU weekly report 2024-10-02
+
+## Work completed since last report
+
+[SiFive function performance report](https://drive.google.com/file/d/1i6RLfZISECriwIesDVeZ9x1TjZHoOO5w).
+
+- WP2
+  - Added inlining of tb functions in do_vsetvl.
+  - Compacted loop of the first Embecosm patch to 8 byte per iteration.
+  - Complete SiFive library function set.
+    - **Complete**
```

Comment:
New bullet
- full results not available, due to some vector instructions triggering SIGILL with VLEN=1024 and LMUL=m8 (under investigation).